app/src/main/java/com/monkeyteam/chimpagne/ui/EventScreen.kt

```diff
@@ -205,59 +206,52 @@ fun EventScreen(
         if (uiState.id.isEmpty()) {
           SpinnerView()
         } else {
-          LazyColumn(
+
+          Column(
               modifier =
                   Modifier.fillMaxSize()
                       .padding(innerPadding)
-                      .background(MaterialTheme.colorScheme.background),
-              verticalArrangement = Arrangement.Top) {
-                item {
-                  Log.d("EventScreen", "ImageUri: $uiState")
-                  ImageCard(uiState.imageUri)
-                }
-                item {
-                  Row(
-                      modifier =
-                          Modifier.horizontalScroll(rememberScrollState()).testTag("tag list")) {
-                        uiState.tags.forEach { tag -> EventTagChip(tag) }
+                      .background(MaterialTheme.colorScheme.background)
+                      .verticalScroll(rememberScrollState())) {
+                ImageCard(uiState.imageUri)
+                Spacer(modifier = Modifier.height(16.dp))
+                Row(
+                    modifier =
+                        Modifier.horizontalScroll(rememberScrollState()).testTag("tag list")) {
+                      uiState.tags.forEach { tag -> EventTagChip(tag) }
+                    }
+                Spacer(modifier = Modifier.padding(vertical = 16.dp))
+                EventMainInfo(event = eventViewModel.buildChimpagneEvent())
+                ChimpagneLogoDivider(
```

Comment:
The dividers are inserted between each item. A more systematic approach with the help of a function would help shortening this.

Reply:
thanks for the tip

Reply:
😽